CommonTools/PileupAlgos/src/PuppiContainer.cc

```diff
@@ -47,11 +47,7 @@ void PuppiContainer::initialize(const std::vector<RecoObj> &iRecoObjects) {
         // float nom = sqrt((fRecoParticle.m)*(fRecoParticle.m) + (fRecoParticle.pt)*(fRecoParticle.pt)*(cosh(fRecoParticle.eta))*(cosh(fRecoParticle.eta))) + (fRecoParticle.pt)*sinh(fRecoParticle.eta);//hacked
         // float denom = sqrt((fRecoParticle.m)*(fRecoParticle.m) + (fRecoParticle.pt)*(fRecoParticle.pt));//hacked
         // float rapidity = log(nom/denom);//hacked
-	if (edm::isFinite(fRecoParticle.rapidity)){
```

Comment:
please recover this isFinite check.
I'm OK if some other "random" is picked instead of ``` reset_PtYPhiM(0, 99., 0, 0) ``` or if it can be skipped,
but we should not let the pseudojet to crash on nans again